Cargo.toml

```diff
@@ -42,7 +42,7 @@ signatory-ledger-tm = { version = "0.11", optional = true }
 subtle-encoding = { version = "0.3", features = ["bech32-preview"] }
 tendermint = { version = "0.3.0-beta1", path = "tendermint-rs" }
 tiny-bip39 = "0.6"
-yubihsm = { version = "0.22.0-alpha2", features = ["setup", "usb"], optional = true }
+yubihsm = { version = "0.22", features = ["setup", "usb"], optional = true }
```

Comment:
Bumped this to the final release (no changes from v0.22.0-alpha2 other than the version bump):

https://github.com/tendermint/yubihsm-rs/pull/194